crates/interpreter/src/host.rs

```diff
@@ -56,3 +51,13 @@ pub trait Host {
     /// Mark `address` to be deleted, with funds transferred to `target`.
     fn selfdestruct(&mut self, address: Address, target: Address) -> Option<SelfDestructResult>;
 }
+
+/// Represents the result of an `sstore` operation.
+#[derive(Debug, Clone, PartialEq, Eq)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+pub struct SStoreResult {
```

Comment:
not sure if this is the right file to place it

Reply:
The place is fine, would add more comments.

original: `Value of the storage when it is first read`
present: `Current value of the storage`
new_value: `Values that that is set`
is_cold: `Is storage slot loaded from database`

Reply:
updated!